torch/_inductor/fx_passes/post_grad.py

```diff
@@ -566,6 +566,11 @@ def cat_noop(inputs, dim=0):
     return len(inputs) == 1
 
 
+@register_noop_decomp(aten.view)
+def view_noop(arg, size):
+    return arg.shape == size
```

Comment:
Well, it's already applied in the first pattern matcher pass (on the joint graph) - this is just reapplying it in case it shows up again (or in inference).